src/ansys/fluent/core/workflow.py

```diff
@@ -1253,8 +1259,11 @@ def _makeTask(command_source, name: str) -> BaseTask:
         "Conditional": ConditionalTask,
     }
     task_type = task.TaskType()
-    if task_type is None and command_source._compound_child:
-        kind = CompoundChild
+    if task_type is None:
+        if command_source._compound_child:
+            kind = CompoundChild
+        else:
+            kind = SimpleTask
```

Comment:
This is done currently to fix a fluent side bug while inserting new tasks. Later this fix can be replaced by a more accurate check of state difference.